app/uk/gov/hmrc/brm/utils/BirthRegisterCountry.scala

```diff
@@ -26,6 +26,8 @@ object BirthRegisterCountry extends Enumeration {
   type BirthRegisterCountry = Value
   val ENGLAND = Value("england")
   val WALES = Value("wales")
+  val NORTHERN_IRELAND = Value("Northern Ireland")
```

Comment:
@manishsw @adamconder If we made it an int would the consuming service not have to know what each value country mapped to? Or do you mean for internal use only?